src/Finch_Run.hpp

```diff
@@ -110,13 +110,13 @@ class Layer
 
     auto writeSolidificationData() { return solidification_data_.write(); }
 
-    std::array<double, 3> getLowerSolidificationDataBounds()
+    std::array<double, 3> getLowerSolidificationDataBounds( MPI_Comm comm )
     {
-        return solidification_data_.getLowerBounds();
+        return solidification_data_.getLowerBounds( comm );
     }
-    std::array<double, 3> getUpperSolidificationDataBounds()
+    std::array<double, 3> getUpperSolidificationDataBounds( MPI_Comm comm )
```

Comment:
Conflicts with ExaCA/bin/runCoupled.cpp line 54 (which uses a call to this function without input arguments)

Reply:
Fixed